test/coverage-test/fixtures/configs/vitest.config.ts

```diff
@@ -3,4 +3,7 @@ import {  defineConfig } from 'vitest/config'
 
 export default defineConfig({
   plugins: [vue()],
+  optimizeDeps: {
```

Comment:
This shouldn't be necessary 🤔

Reply:
I think it started to be required as soon as I added `setupFiles`. 🤔 
CI and local failed constantly when running `pnpm test` in `test/coverage-test`. Adding this fixed the issue immediately.